framework_tvb/tvb/adapters/analyzers/bct_adapters.py

```diff
@@ -197,7 +204,7 @@ def launch(self, connectivity, **kwargs):
 class ModularityOpCSMU(ModularityOCSM):
     """
     """
-    _ui_name = "Optimal Community Structure and Modularity (Undirected)"
+    _ui_name = "Optimal Community Structure and Modularity (Undirected): Connection matrix"
```

Comment:
Same aa above. Maybe `Compute Optimal Community Structure and Modularity (Undirected) from connection matrix` or just `Optimal Community Structure and Modularity (Undirected)` here, as it does not say anything about the Connectivity, more than we already know...